### DISCUSSION TITLE
How to customize the text for categories of completion?

### DISCUSSION THREAD

**Discussion by kohane27** [marlonrichert/zsh-autocomplete#618] [Q&A]
Hello there. Hope you're doing well. Thank you for creating `zsh-autocomplete`. It's a game changer for terminal users!

## Description

I find it difficult to differentiate between the categories (or types?) of completion:

Kitty:

![2023-07-14-16-24-03](https://github.com/marlonrichert/zsh-autocomplete/assets/57322459/fb4cda06-ca1b-4e5f-a908-4428cc4792aa)

[foot](https://codeberg.org/dnkl/foot):

![2023-07-14-16-11-55](https://github.com/marlonrichert/zsh-autocomplete/assets/57322459/f46398a6-1272-4040-8361-eeff4282a193)

By categories of completion, I mean the headers such as `directory`, `filename`, `alias`, `builtin command`, `external command`, `shell function` etc.

I understand that the color is tied to the user's terminal theme. Is there a configuration option to, for example, make them bold or adding ascii boxes around them?

Thank you again for this amazing tool!

**Comment by marlonrichert** [comment]
```zsh
zstyle ':completion:*:*:descriptions' format '<prompt escape codes>'
```
Documentation:
* [`format` completion style](https://zsh.sourceforge.io/Doc/Release/Completion-System.html#index-format_002c-completion-style)
* [Prompt expansion](https://zsh.sourceforge.io/Doc/Release/Prompt-Expansion.html)
